### PR TITLE
Update README with further installation and usage instructions

### PR DESCRIPTION
**Pull request by robertlove**
This makes it clear that a BOULDER OPAL account is required. It also makes it clear what version of Python is required and recommends using Anaconda to get set up easily.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,23 @@ The Q-CTRL Python Package provides an intuitive and convenient Python interface
 
 ## Installation
 
-```shell
+The Q-CTRL Python Package requires **Python 3.6.4**.
```
Comment (charmasaur) [inline README.md]: Just to check, should this be 3.6.4 or 3.6.4+?

Comment (stevegore) [reply README.md]: Technically, it's >=3.6.4,<3.7

**Comment by robertlove** [comment]
@stevegore if we use 
```bash
pip install qctrl
```
instead of
```bash
$ pip install qctrl
```
then we should make this change across the board.

We should also be consistent in the way we refer to packages. For example, on one line we have:

> the Q-CTRL Python Package

then on the very next line we have:

> the `qctrl` package

Use one OR the other - but choose wisely. If you change it to the second way then you need to make this change across the board - all repos!

Thx :)

**Comment by stevegore** [comment]
> @stevegore if we use
> 
> ```shell
> pip install qctrl
> ```
> 
> instead of
> 
> ```shell
> $ pip install qctrl
> ```
> 
> then we should make this change across the board.
> 
> We should also be consistent in the way we refer to packages. For example, on one line we have:
> 
> > the Q-CTRL Python Package
> 
> then on the very next line we have:
> 
> > the `qctrl` package
> 
> Use one OR the other - but choose wisely. If you change it to the second way then you need to make this change across the board - all repos!
> 
> Thx :)

I've been steadily removing the $ from commands across the board, e.g. this PR here https://github.com/qctrl/python-api-client/pull/14

The reason for using `qctrl` instead of just qctrl is because it's the pip name of the package, while "the Q-CTRL Python Package" is the human-friendly name and hence has no special formatting.

Long story short, I'm trying to be consistent but let me know if it still doesn't look that way :)

**Comment by stevegore** [comment]
Actually that README didn't really back up my argument, but I definitely have been making changes to them across the board.

**Comment by saltysealion** [comment]
I suggest keeping the `$` as the target audience understands what this means. It is common practice. 

@robertlove I think we should be more specific about which python versions are supported. ie 3.6 & 3.7. The package may work with later versions of python 3 but it is untested.

**Comment by stevegore** [comment]
> I suggest keeping the `$` as the target audience understands what this means. It is common practice.

I've been having this discussion with Rob. It makes copy/paste hard and I'm not sure it will confuse people *not* having it. The only standards we can find on it indicate that it should be removed _unless_ there are also output lines shown.

> I think we should be more specific about which python versions are supported. ie 3.6 & 3.7. The package may work with later versions of python 3 but it is untested.

Agree with you but struggled to find plain english to put this in on the Friday. How about "Python 3.6.4 or higher, or Python 3.7"?

**Comment by michaelhush** [comment]
LGTM

**Pull request merged**